.editorconfig

```diff
@@ -0,0 +1,25 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# newline ending every file; no trailing space
```

Comment:
If the comment is no trailing space. trim_trailing_whitespace should be true

Reply:
Done.